File: foundation/foundation 2 month/lesson2/work/ex_10.c
#include <stdio.h>
#include <stdlib.h>
#include <time.h>
int enter()
{
return -20+rand()%(10-(-20)+1);
}
void check(int ms[],int n)
{
	int  cnt=0;
	for(int i=0;i<n;i++)
	{
		if(ms[i]<0)
		{
		cnt++;
		}
	}
	printf("\nManfiy sonlar soni -%d ta",cnt);
}
int main()
{
	srand(time(0));
	int n;
	printf("Elelemntar soni: ");
	scanf("%d",&n);
	int arr[n];
	for(int i=0;i<n;i++)
	{
	arr[i]=enter();
	printf("%d,",arr[i]);
	}
	check(arr,n);
return 0;
}

File: full stack/full stack 1 month/lesson3/classwork/ex_10.js
function check(ms)
{
    let arr=[]
    for(let i=0;i<ms.length;i++)
    {
        let cnt = 0
            for(let j=0;j<ms.length;j++)
            {
                    if (ms[i]==ms[j])
                    {
                        cnt+=1
                    }
            }
            if (cnt==1)
            {
                arr.push(ms[i])
            }
    }
    return arr
}

let sonlar = [4, 2, 7, 2, 9, 4, 5, 7, 1];
console.log(check(sonlar))
